theano/tensor/tests/test_opt.py

```diff
@@ -5433,6 +5433,20 @@ def test_local_flatten_lift():
         assert isinstance(topo[1].op, tensor.Elemwise)
 
 
+class Test_Reshape(unittest.TestCase):
+    def setUp(self):
+        self.mode = compile.mode.get_default_mode()
```

Comment:
Travis fails because of this line. when mode=="FAST_COMPILE" canonical optimizations are not executed. It should be ```self.mode = mode_opt```